<launch>
	<arg name="rviz" default="true" />

  <remap from="velodyne_points" to="/kitti_player/hdl64e"/>
	<node pkg="generalp" type="cornerness" name="cornerness" output="screen"/>
	<node pkg="generalp" type="multimap_NDT" name="multimap_NDT" output="screen" respawn="false">
		<param name="voxel_size" type="double" value="0.01"/>
		<param name="resolution" type="double" value="2.4"/>
		<param name="numIter" type="double" value="15"/>
		<param name="size_z" type="double" value="35"/>
		<param name="size_x" type="double" value="80"/>
		<param name="size_y" type="double" value="80"/>
		<param name="pose_init_r" type="double" value="-1.57079"/>
		<param name="pose_init_p" type="double" value="0"/>
		<param name="pose_init_t" type="double" value="0"/>
		<param name="sensor_range" type="double" value="130.3"/>
		<param name="doMultires" type="bool" value="false"/>
		<param name="do3D" type="bool" value="true"/>
		

	</node>
	<node pkg="velodyne_pointcloud" type="cloud_node" name="cloud_node">
		<param name="calibration" value="$(find anestis_loam_mod)/config/vlp16.yaml"/>
		<param name="min_range" value="0.4"/>
		<param name="max_range" value="135.0"/>
	</node>
	
  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find generalp)/rviz_cfg/loam_ndt.rviz" />
  </group>
<param name="use_sim_time" value="true" />

</launch>
